frontend/src/metabase/query_builder/components/EditableText/EditableText.styled.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from "@emotion/styled";
+import { color, alpha } from "metabase/lib/colors";
+import { css } from "@emotion/react";
+
+const sharedStyle = css`
```

Comment:
Somewhere in here we should have `color: ${color("text-dark")};` since our default "black" for text isn't `#000000`